npms/skale-observer/observerWorker.mjs

```diff
@@ -273,7 +294,12 @@ class ObserverServer extends SocketServer {
                         "\n" );
                 }
                 await self.periodicCachingDoNow(
-                    socket, secondsToReDiscoverSkaleNetwork, strChainNameConnectedTo );
+                    socket,
+                    secondsToReDiscoverSkaleNetwork,
+                    strChainNameConnectedTo,
+                    isForceMultiAttemptsUntilSuccess ? true : false
```

Comment:
I think we can remove `? true : false`

Reply:
Done, replaced everywhere it was possible with !! as D1 suggested